README.md

```diff
@@ -50,7 +50,9 @@ happily rely on it for `Span` propagation. To start a new (blocking child)
         ...
         span, ctx := opentracing.StartSpanFromContext(ctx, "operation_name")
         defer span.Finish()
-        span.LogEvent("xyz_called")
+        span.LogFields(
+            log.String("event", "xyz called"),
+            log.Int("request.size", request.Size()))
```

Comment:
Minor nitpick, but from a user's POV, this specific example (`request.size`) is confusing because it looks more like a tag, not something that should have a timestamp.